Lombiq.Tests.UI/Extensions/TenantsUITestContextExtensions.cs

```diff
@@ -49,7 +49,7 @@ public static class TenantsUITestContextExtensions
 
         if (!string.IsNullOrEmpty(featureProfile))
         {
-            await context.SetDropdownByTextAsync("FeatureProfile", featureProfile);
+            await context.ClickReliablyOnAsync(By.XPath("//option[@value='Features Guard']"));
```

Comment:
This extension method is about creating a tenant in Orchard, and is independent of any module of ours. It shouldn't include a reference to Features Guard.

Reply:
Thank you it is a good catch I addressed it be making it flexible.